libs/ml/examples/distributed_learning/distributed_learning_client.hpp

```diff
@@ -17,6 +17,7 @@
 //
 //------------------------------------------------------------------------------
 
+#include "../../examples/distributed_learning/coordinator.hpp"
```

Comment:
don't use relative paths in an include - you just need:
#include "coordinator.hpp"